app/controllers/mixins/breadcrumb_mixin.rb

```diff
@@ -0,0 +1,151 @@
+module Mixins
+  module BreadcrumbMixin
+    def self.included(c)
+      c.helper_method(:data_for_breadcrumbs)
+    end
+
+    def data_for_breadcrumbs
+      if defined?(breadcrumbs_options)
+        breadcrumbs = breadcrumbs_options[:breadcrumbs]
+        record_info = breadcrumbs_options[:record_info] unless breadcrumbs_options[:record_info].nil? || breadcrumbs_options[:record_info]["id"].nil?
+        show_list_title = breadcrumbs_options[:show_list_title]
+        include_record = breadcrumbs_options[:include_record]
+      end
```

Comment:
You're calling a single method 6 times, what about just doing it once?
```
options = breadcrumb_options
```
And then you can refer to it instead of allocation a lot of variables.